dawn/src/dawn/CodeGen/ASTCodeGenCXX.cpp

```diff
@@ -103,6 +103,8 @@ void ASTCodeGenCXX::visit(const std::shared_ptr<iir::IfStmt>& stmt) {
   }
 }
 
+void ASTCodeGenCXX::visit(const std::shared_ptr<iir::LoopStmt>& stmt) { dawn_unreachable("TODO"); }
```

Comment:
do we want to solve this?

Reply:
Maybe eventually. Currently, the only loop descriptor that is implemented works for unstructured meshes only. Still, removed this and deactivated the code gen for this concept the same way we did for the reduction (by declaring an empty body in the header).

Reply:
sounds good!